.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
   - phing recall -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
   # Execute database updates, if there are any.
-  - drush updatedb --yes
+  - drupal update:execute
```

Comment:
If this is really necessary (that is, drush doesn't handle caches properly) then we should update UPDATE.md too and explain why you *must* use DrupalConsole.